core/server/master/pom.xml

```diff
@@ -110,6 +110,11 @@
       <artifactId>alluxio-job-client</artifactId>
       <version>${project.version}</version>
     </dependency>
+    <dependency>
+      <groupId>org.alluxio</groupId>
+      <artifactId>alluxio-stress-shell</artifactId>
```

Comment:
You involve this module can involve hadoop-common module together as hadoop-common is the dependency of `alluxio-stress-shell`.

Reply:
missing `<scope>test</scope>`?